Automattic/wp-calypso
# Editor: Remove override code for "Convert to blocks" button

#### Changes proposed in this Pull Request

* Revert changes from #43902 (remove apps/wpcom-block-editor/src/wpcom/features/convert-to-blocks-button.js since it's now [in Core](https://github.com/WordPress/gutenberg/pull/23704))

#### Testing instructions


Patch to test: D50365-code
Nothing should change with this PR. The "Convert to blocks" button should still show up in the block toolbar:
<img width="657" alt="Screen Shot 2020-09-10 at 11 57 50 AM" src="https://user-images.githubusercontent.com/1689238/92758602-e43d8800-f35c-11ea-9a26-dbd96a92b708.png">

* Verify that Simple and Atomic sites still have the "Convert to blocks" button in the Classic block (from Core Gutenberg).
* The "Convert to blocks" button works as expected.

Fixes #45520
